src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -26,11 +26,10 @@ namespace Tizen.NUI
     /// <summary>
     /// DragAndDrop controls the drag objet and data.
     /// </summary>
+    /// <since_tizen> 10 </since_tizen>
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2000: Dispose objects before losing scope", Justification = "It does not have ownership.")]
-    [EditorBrowsable(EditorBrowsableState.Never)]
     public class DragAndDrop : BaseHandle
     {
-        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Just my curiosity :
Shouldn't we add since_tizen tag for these public handlers?

Reply:
You are right. public delegate also requires <since_tizen>. Please check the follows.
```
        /// <summary>                                                            
        /// Type of callback to render the frame with OpenGLES APIs.             
        /// If the return value of this callback is not 0, the eglSwapBuffers() will be called.
        /// </summary>                                                           
        /// <returns>The return value is not 0, the eglSwapBuffers() will be called.</returns>
        /// <since_tizen> 10 </since_tizen>                                      
        public delegate int GLRenderFrameDelegate(); 
```

Reply:
done